src/main/java/org/oscii/api/LexiconProtocol.java

```diff
@@ -80,15 +93,26 @@ private void addDefinitions(Request request, Response response) {
     }
 
     private void addExamples(Request request, Response response) {
-        List<SentenceExample> results = corpus.examples(request.query, request.source, request.target, request.maxCount, request.memory);
-        // TODO Rank examples (e.g., based on request.context)
-        //  - https://github.com/lilt/core/issues/97
+        long startTime = System.nanoTime(); // - startTime) / 1e9;
+        boolean bHasWord2Vec = word2vecManager.hasModels();
```

Comment:
I think if no word2vec file is specified on the command line, then `word2vecManager` is `null` at this point, so you'll get an exception here. Didn't try it, though.